### ISSUE TITLE
NomenclaturalStatus - curation before uploading first vocabulary version

### ISSUE THREAD

**Issue by ManonGros** [gbif/vocabulary#81]
Here is a file to edit: hhttps://drive.google.com/file/d/1i5i8apmLtIHiBztDltKoNfRXQc7SbrJz/view?usp=sharing

**It contains:**

- the list of existing concepts
- a list of the values already mapped to the concepts (they are all in the **Hidden** sheet/tab for now)
- the GBIF verbatim values for this field that appear more than 10,000 times or in 5 or more datasets

Pease check instructions here: https://github.com/gbif/vocabulary/issues/70

**Comment by CecSve** [comment]
@timrobertson100 or @ManonGros do any of you know where we got the concepts from on this one?

**Comment by timrobertson100** [comment]
From the original [Java enumeration](https://github.com/gbif/gbif-api/blob/master/src/main/java/org/gbif/api/vocabulary/NomenclaturalStatus.java#L31)

```
 * @see <a href="http://dev.e-taxonomy.eu/trac/wiki/NomenclaturalStatus">EDIT CDM</a>
 * @see <a href="http://wiki.tdwg.org/twiki/bin/view/UBIF/LinneanCoreNomenclaturalStatus">TDWG LinneanCoreNomenclaturalStatus</a>
 * @see <a href="http://www.biologybrowser.org/nomglos">Nomenclatural Glossary for Zoology</a>
 * @see <a href="http://www.northernontarioflora.ca/definitions.cfm">Northern Ontario plant database</a>
 * @see <a href="http://rs.gbif.org/vocabulary/gbif/nomenclatural_status.xml">rs.gbif.org vocabulary</a>
 * @see <a href="http://darwin.eeb.uconn.edu/systsem/table.html">Nomenclatural equivalences</a>
```

Please do bear in mind that it was created for the checklistbank (i.e. was meant for taxonomic, regional and other lists of species) rather than for occurrence data, so may not be an appropriate list, or considered useful to try and standardise at all on occurrences. I'd suggest forming an opinion based on the seen values on occurrences.

**Comment by ManonGros** [comment]
https://api.gbif.org/v1/enumeration/basic/NomenclaturalStatus

**Comment by timrobertson100** [comment]
Just in case it's not known - that [API](https://api.gbif.org/v1/enumeration/basic/NomenclaturalStatus) is a representation of the [Java enum](https://github.com/gbif/gbif-api/blob/master/src/main/java/org/gbif/api/vocabulary/NomenclaturalStatus.java#L31). All (ha!) our enums have that.

I had understood the question more as - how did those concepts come about in the first place - and the list of reference sites from the enum documentation would be the likeliest with a few additions over time. This one was started circa 2008.

**Comment by CecSve** [comment]
Yes it was a question on how those concepts were chosen, because the verbatim values really does not translate into those concepts - but that is probably because, as you said, the were made for the checklistbank and not occurrences. The verbatim values more translate into how the name was assigned or the certainty of the name giving

**Comment by ahahn-gbif** [comment]
I would also have expected a nomenclatural status with checklist data (relating to the names as such, not to the way that names apply to occurrences). That being the case, the verbatim data mapped to this concept are possibly there due to a misunderstanding of the term, rather than needing to be handled in our interpretation of this concept. For occurrence data, I would expect the concept of identificationQualifier (cf., aff., possibly including s.str.(?)) to be way more relevant.

**Comment by CecSve** [comment]
Typing this so we remember the feedback from the nomenclature people: 

Sensu stricto and sensu lato (examples of verbatim values) etc would generally also not be used in a checklist setting. Sensu stricto (in strict sense) is used for subgenera to define the sub genera that is named after the genera (which there always is one genus that is), e.g. _Carabus carabus_. Sensu lato is always in context of something that needs to be defined - so the name usage is the broad definition of the species in a given context, so this is typically used in publications and is meaningless in this field if we do not get the context of the 'broad sense' understanding of the name.
